src/main/java/uk/gov/cshr/civilservant/domain/Grade.java

```diff
@@ -22,6 +22,8 @@
     @Column(nullable = false)
     private String name;
 
+    private Boolean core = Boolean.FALSE;
```

Comment:
INFO - This core flag bubbling up from my incorrect structure I think. Every grade should have an Organisation and should be Nullable. Core grades would have Null Organisation. What do you think?

Reply:
Switched to use relationship rather than flag.